alephmarcreader/abstractalephmarcreader.py

```diff
@@ -232,6 +257,27 @@ def get_mentioned_person(self):
         return mentioned
     get_mentioned_person.__annotations__ = {'return': [Person]}
 
+    def get_standardized_date(self):
+        """
+        Returns the standardized date.
+        :return: [StandardizedDate]
+        """
+        std_date = []
+        for field in self.__get_field('046'):
+            date_start = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'c'), '046', 'c')
+            date_end = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'e'), '046', 'e')
+
+            if date_end:
+                date_end = date_end.replace(".", "-")
```

Comment:
You could make `replace(".", "-")` a private method.

Reply:
I did not define the `replace()` method, that's the built-in `str.replace()`.  
Is there an advantage to defining a private method `__replace()`, that will do the very same?

Reply:
I just thought it would be good to have a method that contains `replace(".", "-")`

Reply:
so you wouldn't have to write it twice

Reply:
like this?